README.md

```diff
@@ -1,99 +1,78 @@
 
 The whole project is organized into the following directories:
 
-- `kv`: implementation of the TinyKV key/value store.
-- `proto`: all communication between nodes and processes uses Protocol Buffers over gRPC. This package contains the protocol definitions used by TinyKV, and the generated Go code that you can use.
-- `raft`: implementation of the Raft distributed consensus algorithm, which is used in TinyKV.
-- `scheduler`: implementation of the TinyScheduler which is responsible for managing TinyKV nodes and generating timestamps.
-- `log`: log utility to output log based on level.
-
-### Course material
+* `kv` contains the implementation of the key-value store.
+* `raft` contains the implementation of the Raft consensus algorithm.
+* `scheduler` contains the implementation of the TinyScheduler, which is responsible for managing TinyKV nodes and generating timestamps.
+* `proto` contains the implementation of all communication between nodes and processes uses Protocol Buffers over gRPC. This package contains the protocol definitions used by TinyKV, and the generated Go code that you can use.
+* `log` contains utility to output log based on level.
 
-Please follow the course material to learn the background knowledge and finish code step by step.
+## Reading List
 
-- [Project1 - StandaloneKV](doc/project1-StandaloneKV.md)
-- [Project2 - RaftKV](doc/project2-RaftKV.md)
-- [Project3 - MultiRaftKV](doc/project3-MultiRaftKV.md)
-- [Project4 - Transaction](doc/project4-Transaction.md)
+We provide a [reading list](doc/reading_list.md) for the knowledge of distributed storage system. Though not all of them are highly related with this course, they can help you construct the knowledge system in this field.
 
-## Deploy to a cluster
+Also, you're encouraged to read the overview of TiKV's and PD's design to get a general impression on what you will build:
 
-After you finish the whole implementation, it becomes runnable. You can try TinyKV by deploying it onto a real cluster, and interact with it through TinySQL.
+* TiKV, the design of data storage ([English](https://pingcap.com/blog/2017-07-11-tidbinternal1), [Chinese](https://pingcap.com/blog-cn/tidb-internal-1)).
+* PD, the design of scheduling ([English](https://pingcap.com/blog/2017-07-20-tidbinternal3), [Chinese](https://pingcap.com/blog-cn/tidb-internal-3)).
 
-### Build
+## Build TinyKV from Source
 
-```
-make
-```
+### Prerequisites
 
-It builds the binary of `tinykv-server` and `tinyscheduler-server` to `bin` dir.
+* `git`: The source code of TinyKV is hosted on GitHub as a git repository. To work with git repository, please [install `git`](https://git-scm.com/downloads).
+* `go`: TinyKV is a Go project. To build TinyKV from source, please [install `go`](https://golang.org/doc/install) with version greater or equal to 1.13.
 
-### Run
```

Comment:
I think we can provide the way of running up the cluster, which pinpoints TinyKV can interact with TinySQL

Reply:
Let me try to retain the original Run section in a smooth way.

Reply:
Resolved.